Content.Server/Chemistry/EntitySystems/ChemistrySystem.Injector.cs

```diff
@@ -33,6 +42,38 @@ private void InitializeInjector()
         SubscribeLocalEvent<InjectionCancelledEvent>(OnInjectionCancelled);
     }
 
+    private void AddSetTransferVerbs(EntityUid uid, InjectorComponent component, GetVerbsEvent<AlternativeVerb> args)
+    {
+        if (!args.CanAccess || !args.CanInteract || args.Hands == null)
+            return;
+
+        if (!EntityManager.TryGetComponent<ActorComponent?>(args.User, out var actor))
```

Comment:
There's a `TryGetComp` proxy function you could use instead. It does the same it's just shorthand.

Reply:
Tho it looks like `actor` never gets used so it should probably be `HasComp`.